schemas/reports/facial_similarity_video_report.yaml

```diff
@@ -1,6 +1,11 @@
 allOf:
   - $ref: report_common.yaml
+  - type: object
+    properties:
+      name:
+        type: string
+        enum: [facial_similarity_video]
   - type: object
     properties:
       breakdown:
-        $ref: facial_similarity_video_breakdown.yaml
+        $ref: facial_similarity_video_motion_breakdown.yaml
```

Comment:
Is this correct? Supposedly `facial_similarity_video` is different from motion

Reply:
Yeah, it has been a tentative to reuse same breakdown for both video and motion reports, as from what I've seen the breakdown for the two should follow same structure. Anyway we can have the same file twice (once per video and once per motion) if that makes things clearer.

Reply:
The generate object if we reuse will look a bit odd if it doesn't match the name of the report